openfl/_internal/renderer/cairo/CairoTextField.hx

```diff
@@ -33,6 +33,11 @@ class CairoTextField {
 		
 		var textEngine = textField.__textEngine;
 		var bounds = textEngine.bounds;
```

Comment:
I was a little unsure if I should actually `clone()` the `textEngine.bounds`, but it renders fine. So I think it's OK.